cmd/background-newdisks-heal-ops.go

```diff
@@ -76,10 +84,13 @@ func getLocalDisksToHeal() (disksToHeal Endpoints) {
 			}
 			// Try to connect to the current endpoint
 			// and reformat if the current disk is not formatted
-			_, _, err := connectEndpoint(endpoint)
+			disk, _, err := connectEndpoint(endpoint)
 			if errors.Is(err, errUnformattedDisk) {
 				disksToHeal = append(disksToHeal, endpoint)
 			}
+			if disk != nil && disk.Healing() {
```

Comment:
If we are returning a healing disk in getLocalDisksToHeal() won't be just double healed ?

Reply:
Yes I need to fix this it is double